### PR TITLE
General fixes to es.js

### PR DESCRIPTION
**Pull request by Aby3kr**
- Added and removed some accents
- Fixed some spanish expressions
- Fixed some poorly translated lines and translated some unfinished lines

### CODE REVIEW

**Comment by Aby3kr** [comment]
Quiero dejar claro que se que Dye = Tinte o Colorante pero creo que estos no representan bien el significado de este objeto en krunker

Así que he elegido Dye = Traje ya que es más preciso a mi parecer

**Comment by johnaccardi** [comment]
**Dye** = Traje could be confusing with the Body option
image

The image above shows what it looks more alike to a "**Traje**".
Dye is the colour of the body, so it could be better if is called "**Tinte**" o "**Colorante**", or something else.

**Comment by Aby3kr** [comment]
> **Dye** = Traje could be confusing with the Body option image
> 
> The image above shows what it looks more alike to a "**Traje**". Dye is the colour of the body, so it could be better if is called "**Tinte**" o "**Colorante**", or something else.

Lo decía porque si te das cuenta, lo que cambia realmente es el color de la vestimenta del personaje, en algunos casos llega a cambiar al personaje completamente. Hay una opción para usuarios premium la cual te permite cambiar el color de la piel a tu gusto así que no quería juntar ambos conceptos poniéndole "tinte"

image

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 63: cambiar "Eliminado por" a te eliminó

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 261: Corregir "Precision" a "Precisión"

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 829: Añadir "de cuenta" después de "tu suspención" y corregir "suspención"

**Comment by johnaccardi** [comment]
> > **Dye** = Traje could be confusing with the Body option image
> > The image above shows what it looks more alike to a "**Traje**". Dye is the colour of the body, so it could be better if is called "**Tinte**" o "**Colorante**", or something else.
> 
> Lo decía porque si te das cuenta, lo que cambia realmente es el color de la vestimenta del personaje, en algunos casos llega a cambiar al personaje completamente. Hay una opción para usuarios premium la cual te permite cambiar el color de la piel a tu gusto así que no quería juntar ambos conceptos poniéndole "tinte"
> 
> image

De hecho, poniendome a revisar un poco **Krunker** me he dado cuenta, qué estas cosas no se podrian confundir.
Debido a qué la opción para la gente con **Premium** es "Color de Piel" no Tinte.
image

Por lo tanto, si se podría traducir "Dyes" como "Tintes".

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 969: Cambiar urgentemente "Se tiene que" por  "Debe" 😡

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 834: corregir a "está"

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 181 - Cambiar "viejo" a recorrido"
Linea 186 - Propongo cambiar "Esconder" por "ocultar"

**Review by ivansoadictoalaspatatas — COMMENTED** [review]
Linea 610: Te dejaste tienda

**Comment by Aby3kr** [comment]
Thanks for everything @ivansoadictoalaspatatas @johnaccardi , i think that i have covered every single line (checked all of them 4-5 times) tell me if you find any mistakes

**Review by ivansoadictoalaspatatas — APPROVED** [review]

**Comment by ivansoadictoalaspatatas** [comment]
Me encanta

**Comment by johnaccardi** [comment]
> Thanks for everything @ivansoadictoalaspatatas @johnaccardi, i think that i have covered every single line (checked all of them 4-5 times) tell me if you find any mistakes

Line 600: Spins, it should be "Ruletas"
Line 611 Spins, it should be "Ruletas"

**Comment by Aby3kr** [comment]
> > Thanks for everything @ivansoadictoalaspatatas @johnaccardi, i think that i have covered every single line (checked all of them 4-5 times) tell me if you find any mistakes
> 
> Line 600: Spins, it should be "Ruletas" Line 611 Spins, it should be "Ruletas"

Done, mb

**Review by ivansoadictoalaspatatas — DISMISSED** [review]
Awebo

**Comment by xlNightmare** [comment]
Push Final hecho, a la espera de cierre por @mty22 

Final Push is done, waiting for @mty22

**Review by xlNightmare — DISMISSED** [review]
Changes done, ready for push

**Comment by Aby3kr** [comment]
Thanks to everyone who helped 

IGN: Aby3

**Comment by ivansoadictoalaspatatas** [comment]
IGN: ivanzoo

**Comment by johnaccardi** [comment]
> Gracias @ivansoadictoalaspatatas @johnaccardi @Aby3kr por su contribución con el proyecto de traducción, si pueden dejenme sus IGN Abajo y se les enviara una compensación por su Ayuda.
> 
> 
> 
> - Nightz

Thanks for reviewing the pull request, my IGN is johnaccardi, also I have a question, what's going to happen with the pull-request I made? @xlNightmare

**Comment by xlNightmare** [comment]
@johnaccardi if the changes you made in the other pull request are the same as this one, its probably going to be end up getting closed, and this one will be the last change, unless theres some other changes you made in the other pull request and that didnt get added to this one.

**Comment by johnaccardi** [comment]
> @johnaccardi if the changes you made in the other pull request are the same as this one, its probably going to be end up getting closed, and this one will be the last change, unless theres some other changes you made in the other pull request and that didnt get added to this one.

I think there are some other changes that aren't here, besides that, why mine will be closed if it was first opened, I've been waiting for 2 weeks, this one have been opened for a day. @xlNightmare

**Comment by xlNightmare** [comment]
Sometimes, pull requests dont get to me, or i dont normally see them, sorry for the inconvenient, if you want, you are free to write the changes you think that should be in this one, and i'll review them and will both close your PR and this PR,

Again, sorry for the inconvenience mate

**Comment by johnaccardi** [comment]
> Sometimes, pull requests dont get to me, or i dont normally see them, sorry for the inconvenient, if you want, you are free to write the changes you think that should be in this one, and i'll review them and will both close your PR and this PR,
> 
> Again, sorry for the inconvenience mate

It's all right mate, i like the idea tho, i'll write under this line the changes that should be made in this one to be fully completed,
and ready to merge.

Line 917: Die {0} Veces / should be "Muere {0} Veces"
Line 207 - 215 and 729: Sólo / should be "Solo" without the accent.
Line 834: "Este spin ..." / should be "Esta ruleta ..."
Line 597: "Unirse a clan" / should be "Unirse al clan"
Line 459: "No se han encontrado partidas a la que unirse..." / should be "No se han encontrado partidas a las qué unirse..."

**Comment by mty22** [comment]
ping when good to merge

File: es.js [hunk 1]
```diff
@@ -808,33 +808,33 @@ social.market.sort2 Ordenar
...
 limited.purchase.ask Estás seguro de comprar [{0}] por {1} KR?
 limited.purchase.success Añadido {0} KR A tu Inventario.
 pretend.exit Presiona {0} De nuevo para Salir.
 settings.loaded Configuraciones Cargadas
-settings.loaded.none No se encontro ninguna configuracion
+settings.loaded.none No se encontro ninguna configuración
 account.login.required Necesitas iniciar Sesión
 account.banned.hack Esta cuenta ha sido suspendida permanentemente por Trampas.
-account.banned.appeal Si quieres apelar tu suspención, Contacta:
+account.banned.appeal Si quieres apelar la suspención de tu cuenta , Contacta:
```
Comment (mty22) [inline es.js:829]: `cuenta , Contacta` is the space intended before the comma?

**Comment by johnaccardi** [comment]
I left a couple of comments in the file, things to update. 

Line 829 and 837.
@Aby3kr

**Comment by xlNightmare** [comment]
Alright due to me having some time conveniences i just want to check up with both of you and see if everythinf is ready and is good to merge 

@Aby3kr @johnaccardi

**Comment by johnaccardi** [comment]
> Alright due to me having some time conveniences i just want to check up with both of you and see if everythinf is ready and is good to merge 
> 
> 
> 
> @Aby3kr @johnaccardi 

Thanks for your time to at least review the PR, I really appreciate it, to be honest I was a bit mad cause I tried to contact you but I couldn't, I thought u were never going to review the PR, besides that, we've checked everything, everything is done and ready.

**Ready to Merge**
- John Accardi

@xlNightmare

**Comment by Aby3kr** [comment]
> Alright due to me having some time conveniences i just want to check up with both of you and see if everythinf is ready and is good to merge
> 
> @Aby3kr @johnaccardi

Yes, everything should be ok by now! 

Ready to merge

- Aby3

**Comment by johnaccardi** [comment]
Hay traducción para "Mouse Flick" en línea 102?

@Aby3kr

**Comment by xlNightmare** [comment]
@johnaccardi Transaltor Badges are not given out by myself, you'll for sure receive a KR compensation, although in some point you will be noticed if you are going to be chosen for one.

**Comment by johnaccardi** [comment]
> @johnaccardi Transaltor Badges are not given out by myself, you'll for sure receive a KR compensation, although in some point you will be noticed if you are going to be chosen for one.

All right, cheers for the **KR**, I appreciate it.

**Comment by xlNightmare** [comment]
Alright, thanks to everyone that participated in the Translation project, really hope to see you guys helping in the future and again, thanks for helping!

IGNS:

@Aby3kr : Aby3
@johnaccardi :  johnaccardi
@ivansoadictoalaspatatas : ivanzoo
@xlNightmare : xlNightmare

All good and ready to go @mty22 cheers

**Review by xlNightmare — APPROVED** [review]

**Review by johnaccardi — APPROVED** [review]

**Comment by mty22** [comment]
this is a very good PR

**Comment by mty22** [comment]
this is a very good PR

**Comment by mty22** [comment]
@xlNightmare can you please have a look at the merge conflicts? probably 2 minute job :)

**Comment by mty22** [comment]
ty ty!

**Comment by xlNightmare** [comment]
<3

**Pull request merged**